codesystems/CodeSystem-England-MessageReasonBars.xml

```diff
@@ -0,0 +1,37 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="England-MessageReasonBars" />
+    <url value="https://fhir.nhs.uk/CodeSystem/message-reason-bars" />
+    <version value="1.0.0" />
+    <name value="EnglandMessageReasonBars" />
+    <title value="England Message Reason Bars" />
+    <status value="draft" />
+    <date value="2024-06-14" />
+    <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+    <description value="Message Reason codes." />
```

Comment:
Suggest: A set of codes that define common messaging reasons for BARS Service Requests.